image_topic: "/camera/rgb/image_rect_color"
cam_info_topic: "/camera/rgb/camera_info"
depth_topic: "/camera/depth_registered/sw_registered/image_rect"
mm_to_meters: false
image_freq: 30
min_depth: 0.01
max_depth: 4.0
max_depth_difference: 0.08
max_points_part: 0.09
odom_freq: 30
odom_topic: "/multi_voxels/odom"

#TF FROM EXTERNAL Odometry TO CAMERA FRAME
T_B_P: [1, 0, 0, 0, 0, 1, 0, 0, 0, 0 ,1 ,0, 0, 0, 0, 1] 
max_num_kfs: 10
kf_rate: 15
